overlord/hookstate/hookmgr.go

```diff
@@ -204,10 +207,11 @@ func (m *HookManager) doRunHook(task *state.Task, tomb *tomb.Tomb) error {
 	if hookExists {
 		output, err := runHook(context, tomb)
 		if err != nil {
+			origErr := err
 			err = osutil.OutputErr(output, err)
 			if hooksup.IgnoreFail {
 				task.State().Lock()
-				task.Errorf("ignoring failure in hook %q: %v", hooksup.Hook, err)
+				task.Errorf("ignoring failure in hook %q: %v\noutput: %q", hooksup.Hook, origErr, err)
```

Comment:
This is not necessarily the output, and this will show the output duplicated when there's no output. Needs to be improved or reverted.